host-controller/src/main/java/org/jboss/as/host/controller/HostControllerEnvironment.java

```diff
@@ -44,7 +46,9 @@
 import org.jboss.as.host.controller.jvm.JvmType;
 import org.jboss.as.host.controller.operations.LocalHostControllerInfoImpl;
 import org.jboss.as.network.NetworkUtils;
+import org.jboss.as.server.ServerEnvironment;
```

Comment:
> I think we could try to avoid incorporating server constants into the Host Controller environment.
> 
> If we are going to manage the stream property as a common property for Host Controllers, managed servers and standalone servers, we could move it up to the parent class `org.jboss.as.controller.operations.common.ProcessEnvironment`. That would remove the need to import `ServerEnvironment` into the `HostControllerEnvironment`

Excellent suggestion.  Makes sense to me.

Reply:
Fixed.

Reply:
thanks!